cpp/velox/operators/functions/RegistrationAllFunctions.cc

```diff
@@ -24,9 +27,21 @@
 #include "velox/functions/sparksql/window/WindowFunctionsRegistration.h"
 
 using namespace facebook;
+using namespace facebook::velox;
+using namespace facebook::velox::functions;
```

Comment:
Please remove, and use a specific namespace in the code.

Reply:
Fixed it.